packages/core/modules-sdk/src/remote-query.ts

```diff
@@ -222,6 +223,10 @@ export class RemoteQuery {
       )
     }
 
+    if (ids?.length && !hasPagination) {
+      options.take = null
+    }
```

Comment:
@carlos-r-l-rodrigues I've added this here as a way to retrieve all the expand relationships to be fetched. 

In this case, store was returning 50 sub resources, but remote query was only fetching 15 which was set at a module level.

Reply:
When querying by primary keys, the modules should remove this restriction anyway.
But that's ok to have it here.

Reply:
Just double check if that should be a pk on the module you are querying.